packages/semantic-ui/src/util.js

```diff
@@ -4,21 +4,61 @@ import React from "react";
  * Extract props meant for semantic UI components from props that are
  * passed to Widgets, Templates and Fields.
  * @param {Object} params
- * @param {Object} params.formContext
- * @param {Object} params.uiSchema
- * @param {Object} params.options
+ * @param {Object?} params.formContext
+ * @param {Object?} params.uiSchema
+ * @param {Object?} params.options
+ * @param {Object?} params.defaultSchemaProps
+ * @param {Object?} params.defaultContextProps
  * @returns {any}
  */
 export function getSemanticProps({
   formContext = {},
   uiSchema = {},
   options = {},
+  defaultSchemaProps = { fluid: true , inverted: false },
+  defaultContextProps= { }
 }) {
+
+   const formContextProps = formContext.semantic;
+   let schemaProps = uiSchema["ui:options"] && uiSchema["ui:options"].semantic;
+   let optionProps = options.semantic;
+   // formContext props should overide other props
+   return Object.assign(
+    {},
+    { ...(defaultSchemaProps && defaultSchemaProps) },
```

Comment:
What does `defaultSchemaProps && defaultSchemaProps` do? Why not just do `defaultSchemaProps`?

Reply:
if it is undefined it wont destructure the props. Its quite a neat approach for null/ undefined checks